build/Dockerfile.base

```diff
@@ -0,0 +1,90 @@
+#
+# Import container dependencies
+#
+FROM stevenmburns/xyce as xyce
```

Comment:
Not sure we want this, but maybe it is okay.

Reply:
Removing it for now since we don't have Xyce integrated into the flow. Still an open question whether Xyce should have its own docker service as well.